doc/sailfish-webview-webviewexperimental.qdoc

```diff
@@ -0,0 +1,181 @@
+width and height of the content rectangle can be combined with the x and y
+positions to determine the top-right, bottom-left, and bottom-right pixels
+of the content being displayed in the viewport.
+
+Note that the contentWidth and contentHeight will differ from the
+viewport width and viewport height only if some zoom-factor is being applied
+to the viewport, or if the content is smaller than the viewport.
+
+For more information about these concepts, please see the related Mozilla
+documentation about
+\l{https://developer.mozilla.org/en-US/docs/Glossary/CSS_pixel}{CSS pixels}
+and
+\l{https://developer.mozilla.org/en-US/docs/Web/CSS/Viewport_concepts}{viewports}.
+
+See also: \l{WebView::contentWidth}, \l{WebView::contentHeight},
+\l{WebView::viewportWidth}, \l{WebView::viewportHeight},
+\l{WebView::scrollableSize}
+
+\section2 WebView::contentWidth
+
+\c{qreal}-type read-only property.
+
+The width of the bounding box rectangle defined by the loaded content
+being displayed by the webview.
+
+See also: \l{WebView::contentRect}
+
+\section2 WebView::contentHeight
+
+\c{qreal}-type read-only property.
+
+The height of the bounding box rectangle defined by the loaded content
+being displayed by the webview.
+
+See also: \l{WebView::contentRect}
+
+\section2 WebView::scrollableSize
+
+\c{QSizeF}-type read-only property.
+
+The geometric size of the content being displayed by the webview,
+in device pixels.
+
+The loaded page content is likely to fill an area much larger than the
+viewport of the webview (thus requiring the user to scroll the viewport in
+order to see content further down the page, etc).
+
+This property contains the total size of the content, in device pixels.
+
+See also: \l{WebView::contentRect}, \l{WebView::scrollableOffset}
+
+\section2 WebView::scrollableOffset
+
+\c{QPointF}-type read-only property.
+
+The (x and y pixel) offset into the scrollable size which describes
+the top-left pixel of the content being displayed by the viewport.
+
+The value of this property is effectively equal to the value of the
+x and y values of the \l{WebView::contentRect} after applying the appropriate
+\l{resolution} scale factor (to map from CSS pixel coordinates to
+device-resolution pixel coordinates).
+
+See also: \l{WebView::contentRect}, \l{WebView::scrollableSize},
+\l{WebView::resolution}
+
+\section2 WebView::resolution
+
+\c{float}-type read-only property
+
+The scaling factor which is applied to the (CSS-pixel) contentRect
+values to map to the appropriate device-resolution pixel coordinates.
+
+The value of this property is equal to the width of the device's screen
+divided by the width of the \l{WebView::contentRect}.  As such, it will update
+when the user zooms in or out (thus changing the width, in CSS pixels, of
+the contentRect), or if the page orientation changes (resulting in a change
+to landscape orientation from portrait orientation, or vice-versa).
```

Comment:
Or if `WebEngineSettings::pixelRatio` is changed?

Reply:
added note about pixelRatio to these docs

Reply:
Great; thank you. Digging around the code, one thing I noticed is that there's no `resolutionChanged` signal for when the value updates. I'm not sure whether this should be fixed or not (it feels like it ought to be, but on the other hand it might also generate a lot of signals when zooming); if it's not to be fixed, I wonder if it should be noted here instead.

Reply:
Interestingly, I did some more digging, and I don't see how pixelRatio can affect the resolution at all..
resolution appears to be calculated as: (pseudocode) contentResoution = QMozWindow::size().width() / contentRect.width;